server/plugin/plugin.go

```diff
@@ -207,3 +213,26 @@ func (p *Plugin) loadTemplates(bundlePath string) error {
 	p.Templates = templates
 	return nil
 }
+
+func (p *Plugin) initUserStatusSyncJob() {
```

Comment:
This is going to get run once for every node. Can we put a comment here to make sure that is known?

Reply:
I started naming methods that are not ready for production more explicitly, `Debug...` for instance (or maybe POC in this case?). 0/5 Intention-revealing names are better than comments.